packages/react-components/react-virtualizer/src/hooks/useVirtualizerMeasure.types.ts

```diff
@@ -0,0 +1,12 @@
+export type IVirtualizerMeasureProps = {
```

Comment:
```suggestion
export type VirtualizerMeasureProps = {
```

Can you do a round and remove all the `I` prefixes?